src/scripts/background/connection/callbacks/ProtocolCallbacks.ts

```diff
@@ -17,10 +20,41 @@ async function addWebpage([url]: [string]): Promise<void> {
   await getMintingRegistrar().maybeTryRegister();
 }
 
+async function isWhitelisted() {
```

Comment:
Consider annotating with return type. You will need to wrap in a promise even though the function is async. `: Promise<boolean>`.